tests/features/apps.feature

```diff
@@ -80,6 +80,7 @@ Feature: Subcommand: dab apps
 
 		Examples:
 			| APP            |
+			| athens         |
```

Comment:
This app provides an http interface that ishmael should be discovering, this app should be tested along with the other web apps to ensure the interface is discovered and displayed to the user.